doc/source/release-notes/Percona-Server-5.6.40-83.2.rst

```diff
@@ -0,0 +1,159 @@
+  Bug fixed :psbug:`3901`.
+
+* A clean-up in |Percona Server| binlog-related code was made to avoid
+  uninitialized memory comparison. Bug fixed :psbug:`3925` (upstream
+  :mysqlbug:`90238`).
+
+* The possibility of a truncated bitmap file name was fixed in |InnoDB| logging
+  subsystem. Bug fixed :psbug:`3926`.
+
+* Temporary file I/O was not instrumented for Performance Schema. Bug fixed
+  :psbug:`3937` (upstream :mysqlbug:`90264`).
+
+* A ``key_block_size`` value was set automatically by the Improved MEMORY
+  Storage Engine, which resulted in warnings when changing the engine type to
+  |InnoDB|, and constantly growing ``key_block_size`` during alter operations.
+  Bugs fixed :psbug:`3936`, :psbug:`3940`, and :psbug:`3943`
+
+* |Percona Server| Debian packages description included reference to
+  ``/etc/mysql/my.cnf`` file, which is not actually present in these packages.
+  Bug fixed :psbug:`2046`.
+
+* A fixes were introduced to remove GCC 8 compilation warnings for the
+  |Percona Server| build, retaining compatibility with old compiler versions,
+  including GCC 4.4. Bugs fixed :psbug:`3950` and :psbug:`4471`.
+
+* A typo in ``plugin.cmake`` file prevented to compile plugins statically
+  into the server. Bug fixed :psbug:`3871` (upstream :mysqlbug:`89766`).
+
+* A ``key_block_size`` value was set automatically by the Improved MEMORY
+  Storage Engine, which resulted in warnings when changing the engine type to
+  |InnoDB|, and constantly growing ``key_block_size`` during alter operations.
+  Bug fixed :psbug:`3936`.
+
+* ``-DWITH_NUMA=ON`` build option was silently ignored by CMake when
+  numa development package was not installed, instead of exiting by error.
+  Bug fixed :psbug:`4487`.
+
+* Variables :variable:`innodb_buffer_pool_populate` and :variable:`numa_interleave` mapped to the upstream `innodb_numa_interleave <http://dev.mysql.com/doc/refman/5.6/en/innodb-parameters.html#sysvar_innodb_numa_interleave>`_ variable in :rn:`5.6.27-75.0`were reverted to their original implementation due to the upstream variant limitations. Bug fixed  :psbug:`3967`.
+
+* :variable:`audit_log_include_accounts` variable did not take effect if
+  placed in ``my.cnf`` configuration file, while still working as intended if
+  set dynamically. Bug fixed :psbug:`3867`.
+
+* Synchronization between between :variable:`innodb_kill_idle_transaction` and
+  :variable:`kill_idle_transaction` system variables was broken. Bug fixed
+  :psbug:`3955`.
+
+* A data drift was possible with global status variables when executing the
+  ``SHOW GLOBAL STATUS`` expression. Bug fixed :psbug:`3951` (upstream
+  :mysqlbug:`90351`).
+
+* ``ALTER TABLE ... COMMENT = ...`` statement caused |TokuDB| to rebuild the
+  whole table, which is not needed, as only FRM metadata should be changed.
+  Bugs fixed :psbug:`4280` and :psbug:`4292`.
+
+* A set of compilation fixes was introduced to make |TokuDB| successfully
```

Comment:
I'd rather have something like "Percona Server 8.0 TokuDB fixes have been backported", or better yet, describe the specific issues the fixes address / @georgelorchpercona